src/common/IOSLauncher.ts

```diff
@@ -86,16 +86,14 @@ export class IOSLauncher {
             deviceUDID = currentSimulatorUDID;
         }
 
-        return IOSUtils.launchSimulatorApp()
-            .then(() => {
-                CommonUtils.updateCliAction(
-                    util.format(
-                        messages.getMessage('startDeviceStatus'),
-                        `${this.simulatorName} (${deviceUDID})`
-                    )
-                );
-                return IOSUtils.bootDevice(deviceUDID);
-            })
+        CommonUtils.updateCliAction(
+            util.format(
+                messages.getMessage('startDeviceStatus'),
+                `${this.simulatorName} (${deviceUDID})`
+            )
+        );
+        return IOSUtils.bootDevice(deviceUDID)
+            .then(() => IOSUtils.launchSimulatorApp())
```

Comment:
I get the part of hoisting `updateCliAction`. Can you explain why flipping the order on `bootDevice` and `launchSimulatorApp`?

Reply:
If you recall, we found the issue where Simulator app would launch and open 2 simulators. The fix was to start the simulator first then the simulator app. We fixed this in other repo so I'm doing the same here before I forget about it.

Reply:
Yes, I remember you mentioned that and I was under the impression that you had already done it. So wanted to check it it was swapped for another reason. It makes sense now that the fix was in the other repo. Thanks!